ucum-java8/src/main/java/systems/uom/ucum/UCUM.java

```diff
@@ -210,20 +210,16 @@ public static UCUM getInstance() {
     // /////////////////////////////////////////////////////////////////////
     // The order of GON and DEGREE has been inverted because GON is defined in
     // terms of DEGREE
-    /**
-     * We deviate slightly from the standard here, to maintain compatibility
-     * with the existing NonSI units. In UCUM, the degree is defined as
-     * <code>PI.multiply(RADIAN.divide(180))</code>.
-     */
-    public static final Unit<Angle> DEGREE = addUnit(Units.DEGREE_ANGLE);
     /** As per <a href="http://unitsofmeasure.org/">UCUM</a> standard. */
-    public static final Unit<Angle> GRADE = addUnit(Units.DEGREE_ANGLE.multiply(0.9));
+    public static final Unit<Angle> DEGREE = addUnit(new ProductUnit<Angle>(PI.multiply(RADIAN.divide(360))));
```

Comment:
Although it was `180` in the javadoc, it's divided by `360` in UCUM documentation